03-attribute-operations.Rmd

```diff
@@ -297,7 +297,8 @@ world_continents2 = aggregate(world["pop"], by = ag_var, FUN = sum, na.rm = TRUE
 Combining data from different sources is one of the most common tasks in data preparation. 
 Joins are methods to combine pair of tables based on a shared key variable.
 The **dplyr** package has a set of verbs to easily join `data.frames` - `left_join()`, `right_join()`,  `inner_join()`, `full_join`, `semi_join()` and `anti_join()`.
-The function names follow SQL naming conventions, and @grolemund_r_2016 explain these functions thoroughly in the Relational data chapter in the book R for Data Science .
+
+The function names follow SQL naming conventions, and @grolemund_r_2016 explain these functions thoroughly in the Relational data chapter in the book R for Data Science.
```

Comment:
This single sentence paragraph seems a bit lonely. I suggest removing the preceding line break (deleting L300) and tweaking replacing this line with a slighly modified sentence:

```
These function names follow conventions used in the database language SQL, as explained in [Chapter 13](http://r4ds.had.co.nz/relational-data.html) of *R for Data Science* [@grolemund_r_2016].
```

Shorter and clearer I think.